discord_bot/discord_bot.py

```diff
@@ -1,60 +1,88 @@
+import asyncio
 import json
 import requests
-import asyncio
 import websockets
 
+from .opcodes import Opcodes
+
+
 CONFIG_FILE_PATH = 'configs/config.json'
 DISCORD_GATEWAY_ENDPOINT = 'https://discordapp.com/api/v6/gateway'
 
 
 class DiscordBot:
-
+    """
+    Establishes a connection to the discord gateway and handles varied messages
+    """
     def __init__(self):
-
         # instance variables
         self.config = self.load_config()
         self.gateway_ws_url = self.get_gateway()
+        self.heartbeat_interval = None
```

Comment:
Maybe we should rename this `heartbeat_interval_ms` to make it clear what units it's in

Reply:
Agreed
Done